CondFormats/JetMETObjects/src/SimpleJetCorrector.cc

```diff
@@ -38,7 +38,8 @@ float SimpleJetCorrector::correction(const std::vector<float>& fX,const std::vec
   float result = 1.;
   float tmp    = 0.0;
   float cor    = 0.0;
-  int bin = mParameters.binIndex(fX);
+  int bin = -1;
+  bin = (fX.size()<=3 && fX.size()>0) ? mParameters.binIndexN(fX) : mParameters.binIndex(fX);
```

Comment:
I see. I am sorry I missed "N" at the end. 
Thank you for the details.

Taejeong